gnss_to_map_convertor/launch/gnss_to_map_convertor_launch.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (C) 2022 LEIDOS.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+from ament_index_python import get_package_share_directory
+from launch import LaunchDescription
+from launch_ros.actions import ComposableNodeContainer
+from launch_ros.descriptions import ComposableNode
+from launch.actions import DeclareLaunchArgument
+from launch.substitutions import LaunchConfiguration
+from carma_ros2_utils.launch.get_current_namespace import GetCurrentNamespace
+
+import os
+
+
+'''
+This file is can be used to launch the CARMA gnss_to_map_convertor_node.
```

Comment:
You need to update the carma/launch/localization.launch.py file to launch this node so it will run with the rest of the carma-platform system. 
Also you need to remove the ros1 launch from carma/launch/localization.launch

Once you have updated the files you need to verify the launch occurs correctly and the correct topic connections exist.